### PR TITLE
Feat 41 injection2019

### LINKED ISSUE #41: Map 2019 sharepoint database for Injection
# User story
As a user, I want to map the 2019 sharepoint database, so I can have a more complete BrainInjection model instance

# Acceptance criteria
- [x] Map all known fields for BrainInjection (iacuc_protocol, animal_weight, injection_duration, recovery_time, workstation_id, injection_type, injection_hemisphere, injection_coordinate_ml, injection_coordinate_ap, injection_coordinate_depth, injection_angle)
- [x] map known fields for BrainInjection type Iontophoresis (injection_current, alternating_current)
- [x]  map known fields for BrainInjection type Nanoject (injection_volume)
- [x] handle validation errors

# Sprint Ready Checklist 
- [x] 1. Acceptance criteria defined
- [x] 2. Team understands acceptance criteria
- [x] 3. Team has defined solution / steps to satisfy acceptance criteria
- [x] 4. Acceptance criteria is verifiable / testable
- [x] 5. External / 3rd Party dependencies identified
- [x] 6. Ticket is prioritized and sized

# Notes
*Add any helpful notes here.*

### PR DESCRIPTION
**Pull request by mekhlakapoor**
maps injection 1 and closes #41

### CODE REVIEW

File: tests/sharepoint/resources/list_item1.json [hunk 1]
```diff
@@ -0,0 +1,154 @@
...
+  "Inj2Angle_v2": "0",
+  "FiberImplant1": false,
+  "FiberImplant1DV": "4.2",
+  "FiberImplant2": false,
+  "FiberImplant2DV": "4.2",
+  "ID": 5554,
+  "OData__UIVersionString": "4.0",
+  "Attachments": false,
+  "GUID": "00c3c333-c4a0-42aa-adab-88183c324201"
+}
```
Comment (jtyoung84) [inline tests/sharepoint/resources/list_item1.json:154]: Can you add a new line here?

File: tests/sharepoint/resources/list_item2.json [hunk 2]
```diff
@@ -0,0 +1,154 @@
...
+  "Inj2Angle_v2": "0",
+  "FiberImplant1": false,
+  "FiberImplant1DV": "4.2",
+  "FiberImplant2": false,
+  "FiberImplant2DV": "4.2",
+  "ID": 5554,
+  "OData__UIVersionString": "4.0",
+  "Attachments": false,
+  "GUID": "00c3c333-c4a0-42aa-adab-88183c324201"
+}
```
Comment (jtyoung84) [inline tests/sharepoint/resources/list_item2.json:154]: Can you add a new line here too?

**Review by jtyoung84 — CHANGES_REQUESTED** [review]
A couple of small "add new line" fixes

**Comment by mekhlakapoor** [comment]
curl -X GET http://localhost:5000/procedures/434379
{"message":"Valid Model.","data":{"describedBy":"https://raw.githubusercontent.com/AllenNeuralDynamics/aind-data-schema/main/src/aind_data_schema/procedures.py","schema_version":"0.5.2","subject_id":"434379","headframes":null,"craniotomies":null,"mri_scans":null,"injections":[{"type":null,"start_date":"2019-01-04","end_date":"2019-01-04","experimenter_full_name":"Ali","iacuc_protocol":"1707","animal_weight":null,"anaesthesia":{"type":"isoflurane","duration":"1","level":"1.75"},"notes":null,"injection_materials":null,"recovery_time":15,"injection_duration":5,"workstation_id":"SWS 4","instrument_id":"Ionto #4","injection_hemisphere":"Right","injection_coordinate_ml":3.4,"injection_coordinate_ap":-4.72,"injection_coordinate_depth":2.2,"injection_angle":0.0,"injection_type":"Iontophoresis","injection_current":"5uA","alternating_current":"7/7"}],"fiber_implants":null,"water_restrictions":null,"training_protocols":null,"tissue_preparations":null,"other_procedures":null,"notes":null}}%

**Review by jtyoung84 — APPROVED** [review]

File: src/aind_metadata_service/sharepoint/utils.py [hunk 3]
```diff
@@ -0,0 +1,26 @@
...
+
+from aind_data_schema.procedures import Side
+
+
+def map_hemisphere(response_hemisphere) -> Optional[Side]:
+    """Maps response string to Side object"""
+    if response_hemisphere == Side.LEFT.value:
+        return Side.LEFT
+    else:
+        return Side.RIGHT
```
Comment (dyf) [inline src/aind_metadata_service/sharepoint/utils.py:14]: Not sure we should assume the if it's not Left that it's Right.

**Review by dyf — CHANGES_REQUESTED** [review]
small thing

File: src/aind_metadata_service/sharepoint/utils.py (hunk 3 above)
Comment (mekhlakapoor) [reply src/aind_metadata_service/sharepoint/utils.py:14]: good point. in the request form, the field for hemisphere is a choice between Right/Left but I guess if neither is selected itd be "Select ...", i'll update and add another example to cover that case in unit test

Comment (jtyoung84) [reply src/aind_metadata_service/sharepoint/utils.py:14]: I think you make it None in that case

Comment (mekhlakapoor) [reply src/aind_metadata_service/sharepoint/utils.py:14]: yup thats how i handled it in the latest commit

**Review by dyf — APPROVED** [review]
LGTM

File: src/aind_metadata_service/sharepoint/client.py [hunk 4]
```diff
@@ -360,19 +381,129 @@ def _handle_response_from_sharepoint(  # noqa: C901
...
+            list_item.get_property(list_fields.VIRUS_A_P.value)
+        )
+        # TODO: handle 2 values for depth (for now using 1st value)
+        injection_coordinate_depth = parse_str_into_float(
+            list_item.get_property(list_fields.VIRUS_D_V.value)
+        )
+        injection_angle = parse_str_into_float(
+            list_item.get_property(list_fields.INJ1ANGLE0.value)
+        )
+        notes = list_item.get_property(
```
Comment (saskiad) [inline src/aind_metadata_service/sharepoint/client.py:444]: We should remove this - we aren't going to track surgeon comments

**Pull request merged**